src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -2807,12 +2812,17 @@ private void removeSourcesFromOcto(MediaSourceMap sources, MediaSourceGroupMap s
          */
         private void setRelays(List<String> allRelays)
```

Comment:
The only use of `setRelays` is from `updateOctoRelays`. Should we bubble up the check there?